### DISCUSSION TITLE
Remove underline style in zsh paths

### DISCUSSION THREAD

**Discussion by EliabHdez** [ohmyzsh/ohmyzsh#9690] [Help] [answered]
Can anyone tell me how can I remove the underscore in the paths in zsh? I don't like this underline...

**Comment by mcornella** [comment, marked as answer]
What underline? Can you post a screenshot?

**Reply by EliabHdez** [reply]
Yes, of course! #
![subrayado zsh](https://user-images.githubusercontent.com/77374111/109404221-819b7100-7929-11eb-9aa4-057ad6496e72.JPG)

**Reply by mcornella** [reply]
That's a Windows Terminal problem. See https://github.com/microsoft/terminal/issues/7070 and the original issue linked there.

**Reply by EliabHdez** [reply]
Yes and not! let me explain. The problem in windows terminal is that it lengthens the underline when a new panel is opened. But the underline is an attribute that zsh brings and that I want to remove, I don't like the underline that zsh brings by default. 

Here I am in the terminus terminal and although there is no problem of the lengthening of the underline, this underline is still present in the routes, and this underline is the one that I want to eliminate...
![terminus terminal subrayado en zsh](https://user-images.githubusercontent.com/77374111/109432799-9122c480-79d2-11eb-9757-854b4572fc51.JPG)

**Reply by mcornella** [reply]
Might be a zsh-syntax-highlighting thing? It's difficult to know without knowing your configuration. This is not a default zsh thing.

**Reply by EliabHdez** [reply]
You're right. It's zsh-sintax-highlighting thing. My configuration has activated the plugins sintax highlighting and auto suggestions.  The problem is that in the github of sintax highlighting doesnt say how to remove it. But well, my apologies for believing it was a zsh thing

**Reply by mcornella** [reply]
By default paths are underlined. You need to toggle that off with
```zsh
ZSH_HIGHLIGHT_STYLES[path]='none'
```
[Documentation](https://github.com/zsh-users/zsh-syntax-highlighting/blob/master/docs/highlighters/main.md).

**Reply by EliabHdez** [reply]
Ooooohhh thanks very much. You are big brother. I hope it was not a bother to advise me, but my English is not very good yet and then I do not know where to look for the information in the repositories. But I could already remove the underline thanks for your answer! Thanks 👍🏻
